### PR TITLE
Migrate from Jenkins to GitHub actions

### PR DESCRIPTION
**Pull request by joakimen**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-      // as we do not want credentials to be exposed to the job
-      // on other branches or in PRs.
-      //
-      // To have the correct version applied to the build we need
-      // to use a hook that is run during semantic-release execution.
-      // For this we use the 'prepack' hook. This also ensures
-      // that 'npm link' and alike builds the code, and we can
-      // trigger the same hook on other branches.
-
-      if (env.BRANCH_NAME ==~ /^(master|\d+\.(\d+|x)(\.x)?)$/) {
```
Comment (joakimen) [inline Jenkinsfile:47]: Tok ikke med støtte for det alternative branchnavnet, regner med det dreide seg om f.eks denne branchen, [1.x](https://github.com/capralifecycle/cals-cli/tree/1.x), men den har ikke vært oppdatert på årevis

**Comment by joakimen** [comment]
We need to decide how to handle public Node packages and deal with it here before merging.

**Comment by joakimen** [comment]
> We need to decide how to handle public Node packages and deal with it here before merging.

Token has been created, added reference to it in the publish-stage (as per https://semantic-release.gitbook.io/semantic-release/recipes/ci-configurations/github-actions)

**Review by stekern — APPROVED** [review]
LGTM!

(nit: would've preferred `fix(ci)` instead of `feat` just to reduce the scope of changes to the semantic version, but it's not a big deal)

**Pull request merged**